default/profiles/settings-local.js

```diff
@@ -1 +1,2 @@
 window.spinnakerSettings.feature.managedServiceAccounts = false;
+window.spinnakerSettings.authEnabled = true; # uncomment this if you are using openlap
```

Comment:
# window.spinnakerSettings.authEnabled = true; # uncomment this if you are using saml